src/drivers/mct/cime_config/config_component.xml

```diff
@@ -485,6 +485,19 @@
     </desc>
   </entry>
 
+  <entry id="COMPLETION_SETS_CONTINUE_RUN">
```

Comment:
I don't think we need another variable to do this.   You just need to make sure that the namelists are not updated too early.

Reply:
The other idea I had was to submit another job which would run `xmlchange CONTINUE_RUN=TRUE`, and make the resubmits dependent on that, but that seems worse. Otherwise I don't know of any way of doing this, since case.submit submits all the jobs, but can't set `CONTINUE_RUN` because of the namelist generation in `case.run`, and AFAIK, has no other portable way to tell case.run to set `CONTINUE_RUN` after the namelists are generated.

Reply:
Submitting an extra job that only changes continue_run to true is what I did when I create a script to queue up runs.